templates/image_file/index.html.twig

```diff
@@ -4,13 +4,13 @@
 
```

Comment:
Making a note here that we should probably remove all of the image_file stuff at some point; they're not used and I don't believe there are any plans to do so.